server/Makefile

```diff
@@ -0,0 +1,22 @@
+$(shell cargo install default-target)
+BUILD_DIR ?= build
+NEPTUNE_SERVER_VERSION = v0.1.0
+BUILD_ERR_FILE = build-error.log
+ERR_DIR ?= build-errors
+BUILD_TARGET ?= $(shell default-target)
+
+.PHONY: clean
+clean: ## clean build artifacts
+	rm -r ${BUILD_DIR} 2> /dev/null || :
+	rm -r ${ERR_DIR} 2> /dev/null || :
```

Comment:
what's the reason for the `|| :` here? Could we just `rm -f {dir}`? Just trying to simplify this makefile as much as possible, there's a lot of jargon that isn't the most readable